include/experimental/__p2630_bits/submdspan_mapping.hpp

```diff
@@ -182,6 +182,21 @@ struct deduce_layout_left_submapping<
   }
 };
 
+// We are reusing the same thing for layout_left and layout_left_padded
+// For layout_left as source StaticStride is static_extent(0)
+template<class Extents, size_t NumGaps, size_t StaticStride>
+struct Compute_S_static_layout_left {
```

Comment:
```suggestion
struct compute_s_static_layout_left {
```
following the names of the other structs in this file.

Reply:
Done